server/parsec/backend/realm.py

```diff
@@ -381,32 +329,6 @@ async def api_realm_update_roles(
 
         return authenticated_cmds.latest.realm_update_roles.RepOk()
 
-    @api
-    async def apiv2_realm_start_reencryption_maintenance(
-        self,
-        client_ctx: AuthenticatedClientContext,
-        req: authenticated_cmds.v2.realm_start_reencryption_maintenance.Req,
-    ) -> authenticated_cmds.v2.realm_start_reencryption_maintenance.Rep:
-        # `realm_start_reencryption_maintenance` command is similar between APIv2 and v4+ from the server
-        # point of view.
-        # (from client point of view, server may return `bad_timestamp` response
-        # with some fields missing)
-        # TODO: proper api req/rep conversion
-        return await self.api_realm_start_reencryption_maintenance(client_ctx, req)  # type: ignore[return-value, arg-type]
-
-    @api
-    async def apiv3_realm_start_reencryption_maintenance(
```

Comment:
APIv3 should stay !

Reply:
See https://github.com/Scille/parsec-cloud/pull/4845/files#r1276297985